docs/image-yaml.rst

```diff
@@ -91,10 +91,10 @@ partitions
         (*optional*) Partition offset from the beginning of the image. Offset
         can be used to tweak the position of the first partition.
     size
-        (*optional*) Size of the partition.  If not specified, will be
-        automatically computed based on the size of contents, the partition
-        role, and any limits imposed by offsets specified for partitions
-        located after this one on the disk.
+        (*optional*) Size of the partition.  If not specified, 64M is used for
```

Comment:
This is a special case of 'automatically computed based on size of contents and the partition role'.  -1 for listing this specific value here.